common/handler_env.py

```diff
@@ -0,0 +1,40 @@
+from PIL import Image
+from abc import ABC, abstractmethod
+
+from qdrant_client.grpc import ScoredPoint
+
+from common.consts import PROJECT_PATH
+from metrics.consts import MetricCollections
+
+
+class EnvFunctionHandler(ABC):
+    """
+    Base class for listing shared methods.
+    """
+
+    local_data_dir = PROJECT_PATH / "data"
```

Comment:
You can keep same naming for different implemenations, without local/cloud prefix so you dont have to repeat them in every implemented function

Reply:
Fair point, but I did it on purpose - those variables are inherited by both local and cloud classes and in cloud-based class it is useful to know that these path are supposed to be local.